exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorker.java

```diff
@@ -165,23 +158,25 @@ boolean registerView(View view) {
     }
   }
 
-  // Polls ViewData from Stats library for all exported views, and upload them as TimeSeries to
-  // StackDriver.
+  // Polls MetricProducerManager from Metrics library for all registered metrics, and upload them as
+  // TimeSeries to StackDriver.
   @VisibleForTesting
   void export() {
-    List</*@Nullable*/ ViewData> viewDataList = Lists.newArrayList();
-    for (View view : viewManager.getAllExportedViews()) {
-      if (registerView(view)) {
-        // Only upload stats for valid views.
-        viewDataList.add(viewManager.getView(view.getName()));
+    List</*@Nullable*/ Metric> metricsList = Lists.newArrayList();
```

Comment:
`@Nullable` annotation is no longer needed here, since I don't think `metricProducer.getMetrics()` would return null.